diagnosticsModule/Test/Private/AdfsHealthChecks.Test.ps1

```diff
@@ -104,6 +104,161 @@ InModuleScope ADFSDiagnosticsModule {
         }
     }
 
+    Describe "TestADFSDuplicateSPN" {
+        
+        BeforeAll {
+            $_upnServiceAccount = "aadcsvc@contoso.com"
+            $_samServiceAccount = "contoso\aadcsvc"
+            $_localSystemAccount = "LocalSystem"
```

Comment:
Also add tests where the casing doesn't match.

Reply:
-eq is case insensitive.